### ISSUE TITLE
TypeError: coercing to Unicode: need string or buffer, NoneType found

### ISSUE THREAD

**Issue by vinnitu** [golastmile/rasa_nlu#175]
```
$ python -m rasa_nlu.server -c config_mitie_sklearn.json --server_model_dir=./model_20170223-204523/
/usr/local/lib/python2.7/dist-packages/rasa_nlu-0.6b0-py2.7.egg/rasa_nlu/__init__.py:1: UserWarning: Module rasa_nlu was already imported from /usr/local/lib/python2.7/dist-packages/rasa_nlu-0.6b0-py2.7.egg/rasa_nlu/__init__.pyc, but /home/user/rasa_nlu is being added to sys.path
{
    "num_threads": 1, 
    "log_level": 20, 
    "language": "en", 
    "mitie_file": "./data/total_word_feature_extractor.dat", 
    "emulate": null, 
    "config": "config_mitie_sklearn.json", 
    "write": "/home/user/rasa_nlu/rasa_nlu_log.json", 
    "token": null, 
    "server_model_dir": "./model_20170223-204523/", 
    "path": "./", 
    "log_file": null, 
    "data": "./data/sveta.json", 
    "port": 5000, 
    "backend": "mitie_sklearn"
}
INFO:root:using mitie_sklearn backend
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "build/bdist.linux-x86_64/egg/rasa_nlu/server.py", line 259, in <module>
  File "build/bdist.linux-x86_64/egg/rasa_nlu/server.py", line 21, in __init__
  File "build/bdist.linux-x86_64/egg/rasa_nlu/server.py", line 59, in __create_interpreter
  File "build/bdist.linux-x86_64/egg/rasa_nlu/interpreters/mitie_sklearn_interpreter.py", line 18, in __init__
TypeError: coercing to Unicode: need string or buffer, NoneType found
```

**Comment by amn41** [comment]
Looks like it can't find the `.pkl` for the intent classifier. Could you post the `metadata.json`? Did the training complete successfully?

**Comment by vinnitu** [comment]
files in .//model_20170223-204523$ ls
file list in ./model_20170223-204523
```
$ ls
entity_extractor.dat
index.json
intent_classifier.dat
metadata.json
training_data.json
```

/model_20170223-204523/metadata.json

```
{
    "trained_at": "20170223-204523",
    "intent_classifier": "./model_20170223-204523/intent_classifier.dat",
    "training_data": "./model_20170223-204523/training_data.json",
    "language_name": "en",
    "entity_synonyms": "./model_20170223-204523/index.json",
    "entity_extractor": "./model_20170223-204523/entity_extractor.dat",
    "rasa_nlu_version": "0.6b0",
    "feature_extractor": "./data/total_word_feature_extractor.dat",
    "backend": "mitie_sklearn"
}
```

**Comment by steookk** [comment]
same problem here

**Comment by tmbo** [comment]
should be fixed in latest master (version `0.7.0a1`)
